### PR TITLE
Allow missing conf for plugins-extras

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: plugins/torproxy.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 import json
 import io
 import iniparse.compat as ini
+import logging
+
+logger = logging.getLogger('dnf.plugin')
```
Comment (ignatenkobrain) [inline plugins/torproxy.py]: we have logger configured in `dnfpluginsextras`

**Comment by MichaelMraka** [comment]
@dnf-bot r+

**Comment by j-mracek** [comment]
test this please

**Comment by j-mracek** [comment]
test this please

**Comment by j-mracek** [comment]
test this please

**Comment by j-mracek** [comment]
@dnf-bot retry

**Pull request closed without merging**